machina/apps/forum/abstract_models.py

```diff
@@ -144,6 +144,8 @@ def clean(self):
         if self.is_link and not self.link:
             raise ValidationError(_('A link forum must have a link associated with it'))
 
+        return self.cleaned_data
```

Comment:
I don't think that model `clean()` methods should return anything. Actually I'm not event sure that model instances have such `cleaned_data` attribute (look at the [base implementation](https://github.com/django/django/blob/73b1b225ce1a3318d4478f90cc0db0a260aba3aa/django/db/models/base.py#L1004-L1011) for the model `clean()` method - it doesn't return anything).

I agree that form `clean()` methods should return this attribute, but for models it doesn't seem to be the case. 😉

Reply:
Sorry. The one from `form` was the only one that bit me and then I did. a global search to create the PR and did not notice that these are for models. :-)

The PR has been updated.